docs/source/install.rst

```diff
@@ -136,9 +139,9 @@ However, if for any reason you need to force-install a particular CUDA version (
 
     cuDNN, cuTENSOR, and NCCL are available on ``conda-forge`` as optional dependencies. The following command can install them all at once::
 
-        $ conda install -c conda-forge cupy cudatoolkit=10.1 cudnn cutensor nccl
+        $ conda install -c conda-forge cupy cudnn cutensor nccl
 
-    Each of them can also be installed separately as needed. Note that cuTENSOR is available since CUDA 10.1+.
+    Each of them can also be installed separately as needed. Note that cuTENSOR is available since CUDA 10.2+.
```

Comment:
I think this note is no longer necessary as CUDA 10.2 is now the minimum supported version.

```suggestion
    Each of them can also be installed separately as needed.
```

Reply:
Sounds good!